modules/camera/camera_linux.cpp

```diff
@@ -82,6 +83,7 @@ void CameraLinux::_update_devices() {
 
 		usleep(1000000);
 	}
+	emit_signal(SNAME(CameraServer::feeds_updated_signal_name));
```

Comment:
The call should happen just before `usleep`. Otherwise, it will only be called once we stop monitoring feeds.